src/jit/emitxarch.cpp

```diff
@@ -1085,6 +1181,7 @@ inline bool hasCodeMI(instruction ins)
 inline size_t insCodeMI(instruction ins)
 {
     assert((unsigned)ins < _countof(insCodesMI));
+    size_t code = insCodesMI[ins];
```

Comment:
why this change?

Reply:
debug help for conditional breakpoint - to be removed